Sources/CorePayments/Networking/Enums/Environment.swift

```diff
@@ -22,4 +22,13 @@ public enum Environment {
             return URL(string: "https://paypal.com/graphql")!
         }
     }
+    
+    public var toString: String {
```

Comment:
General thought - I think we've used a mix of things in these cases `toString` here, `stringValue` in others, and `description` in some places. Not part of this PR but I wonder if it'd be helpful at some point to audit this before the SDK gets too large. I think on BT as we've done the conversions we've generally used `stringValue` but I don't necessarily feel strongly tied to one name vs any other. It would be great to make this consistent across the SDKs though!

Reply:
Yeh, good callout - that would be nice to standardize. Maybe we can look to Apple's APIs to see which language aligns best with their APIs.